webserver/Models/DTOs/CompanyDTO.cs

```diff
@@ -26,16 +25,11 @@ public class CompanyDTO {
     }
 
     public static explicit operator Company(CompanyDTO dto){
-        return new Company {
-            Id = dto.Id,
-            UserName = dto.UserName,
-            Email = dto.Email,
-            PhoneNumber = dto.PhoneNumber,
-
-            Country = dto.Country,
-            State = dto.State,
-            City = dto.City
-        };
+        return new Company (
+            dto.Id, dto.UserName, dto.Email, dto.PhoneNumber,
+
+            dto.Country, dto.State, dto.City
+        );
```

Comment:
It's not easier to read either.
Better in the previous way.

Reply:
Done. I elected to give a line of separation between IdentityUser fields (from which the User classes inherit from) and the custom fields (Guacamole's)